test/performance/index_performance_test.cpp

```diff
@@ -390,7 +390,7 @@ static void TestIndexPerformance(const IndexType& index_type) {
 }
 
 TEST_F(IndexPerformanceTests, MultiThreadedTest) {
-  std::vector<IndexType> index_types = {INDEX_TYPE_BTREE, INDEX_TYPE_SKIPLIST, INDEX_TYPE_BWTREE};
+  std::vector<IndexType> index_types = {INDEX_TYPE_BTREE, INDEX_TYPE_BWTREE, INDEX_TYPE_BWTREE};
```

Comment:
The duplicated "INDEX_TYPE_BWTREE" could be removed to avoid running the same test twice (we use a vector to hold index types that performance testing will be running for)